source/geod24/LocalRest.d

```diff
@@ -688,6 +688,7 @@ public final class RemoteAPI (API) : API
                         (C.OwnerTerminated e) { terminated = true; },
                         (ShutdownCommand e) {
                             terminated = true;
+                            C.thisTid().shutdowned = true;
```

Comment:
Also, "shutdowned" is not correct in English: the fiber is "shut down", the verb is "shut". So the past tense of "shutdown" is "shutdown".